src/WebJobs.Script/Config/ScriptApplicationHostOptions.cs

```diff
@@ -0,0 +1,27 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System;
+using System.IO;
+using Microsoft.Azure.WebJobs.Script.Config;
+
+namespace Microsoft.Azure.WebJobs.Script
```

Comment:
Should this be in Microsoft.Azure.WebJobs.Script.Configuration namespace?

Reply:
Would like to avoid having to add a namespace just to get the options reference. This would be consistent with other frameworks

Reply:
Agree with that - I made the commend because I thought you had other options types that were in a Config namespace. If not and we're consisten, great :)